sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -88,6 +88,21 @@ To interact with these resources, one should be familiar with the following SDK
 
 For more concepts and deeper discussion, see: [Service Bus Advanced Features](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-messaging-overview#advanced-features).
 
+### Client lifetime
+
+Each of the Service Bus client types is safe to cache and use as a singleton for the lifetime of the application, which is best practice when messages are being published or read regularly or semi-regularly. The clients are responsible for efficient resource management, working to keep resource usage low during periods of inactivity and manage health during periods of higher use. The clients are disposable and require calling `DisposeAsync` is required to ensure that network resources and other unmanaged objects are properly cleaned up.
```

Comment:
> The clients are responsible for efficient resource management, working to keep resource usage low during periods of inactivity and manage health during periods of higher use.

I'm not really sure what this means in this context. Can we be more specific?

Reply:
Tried a bit or wordsmithing.  Does it read better to you now?